rott/modexlib.c

```diff
@@ -35,7 +35,7 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 static void StretchMemPicture ();
 // GLOBAL VARIABLES
 
-boolean StretchScreen=0;//bn�++
```

Comment:
I don't know how to not change this, my editor does not support this character. What do I do?

Reply:
Just remove that character from the comment.

Reply:
Okay, I've changed it now.